modules/web/license-checker-config.json

```
@@ -3,7 +3,7 @@
     "*", ".git", ".angular", ".github", ".idea", "docs", "vendor", ".husky", "dist", "coverage",
     "src/assets/fonts", "src/assets/js", "src/assets/css/font-awesome", "src/app/dynamic/enterprise/**",
     "src/index.html", "**/.DS_Store", "**/.gitignore", "**/*.json", "**/*.yaml", "**/*.svg",
-    "**/*.txt", "**/*.md", "pkg", "cmd", "certs"
+    "**/*.txt", "**/*.md", "pkg", "cmd", "certs", "**/OWNERS"
```

We should also remove API related directories now that its in separate module.

```suggestion
    "**/*.txt", "**/*.md", "**/OWNERS"
```